packages/suite/src/components/guide/GuideButton.tsx

```diff
@@ -23,18 +23,23 @@ const Wrapper = styled.button`
     box-shadow: 0 2px 7px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_15},
         0 2px 3px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_5};
     transition: opacity 0.3s ease 0.3s;
+    opacity: ${({ $isGuideOpen }) => ($isGuideOpen ? 0 : 1)};
+
+    &:focus {
```

Comment:
It's not necessary to use `&` in StyledComponents :)

Reply:
Cool, 167ab5c587e39af124eff83339a824da5b5efbb1